src/fringe.js

```diff
@@ -23,11 +25,11 @@ import {MediaManager} from './mediaManager.js';
 export function LunaticFringe(canvas, hidden, visibilityChange) {
     "use strict";
 
-    var animationLoop, objectManager, mediaManager, Key, DEBUG = false, numEnemiesKilled = 0, score = 0;
-    var game = this;
-	var Version = "1.26";
-	var isCapsPaused = false;
-	log("Game Version: " + Version);
+    // var animationLoop, objectManager, mediaManager, Key, DEBUG = true, numEnemiesKilled = 0, score = 0;
+    // var game = this;
+	let version = "1.26";
```

Comment:
- [ ] Update the version, even though I am really bad at making sure to update it when I push changes